_posts/2018-06-01-another-successful-rails-girls-event.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Another successful Rails Girls event"
+date: 2018-06-01 09:00:00
+categories: ["learning", "rails"]
+author: "luciano"
+---
+
+Last month was the fifth edition of the [Rails Girls Buenos Aires](http://railsgirls.com/buenosaires.html), and third participation by Ombu Labs as a sponsor and part of the coaching team. This time it took place in the impressive offices of Google Argentina where 25 participants attended. All of them were very excited and full of motivation to start learning new skills.
+
+<!--more-->
+
+<img src="/blog/assets/images/rails-girls/everybody.jpg">
+
+For those who never heard about Rails Girls, it’s a 2-day workshop to introduce women to programming, with a focus on the [Ruby on Rails](https://rubyonrails.org/) framework. We also talk about the current situation of women in technology and the things that we can do to increase the number of women working in this industry.
+
+During the workshop, each participant developed a basic Ruby on Rails application from scratch. Most of them had no previous experience in programming but one of the great things about Rails is that it's incredibly friendly for beginners because it allows you to create things without being worried about some advanced concepts that you don't need to know yet.
+
+This event was again a great experience, even as a coach you end up learning a lot from the participants. Here are a couple of photos of that day:
+
+<img src="/blog/assets/images/rails-girls/group1.jpg">
+
+<img src="/blog/assets/images/rails-girls/teaching.jpg">
+
+<img src="/blog/assets/images/rails-girls/group2.jpg">
+
+<img src="/blog/assets/images/rails-girls/mvc.jpg">
+
+<img src="/blog/assets/images/rails-girls/group3.jpg">
+
+<img src="/blog/assets/images/rails-girls/rails-landing.jpg">
+
+In Ombu Labs we believe that it is very important that more companies start having more diverse teams every time. That's why we decided to contribute to this cause, and we’re going to keep doing it.
```

Comment:
"_At_ Ombu Labs we believe that it is very important that more companies start _working to have_ more diverse teams".